### ISSUE TITLE
Revise the Database

### ISSUE THREAD

**Issue by erikperin** [Vialerta/Core#3]
Make a revise of the first database schema...

[vialerta_core_db.mwb.tar.gz](https://github.com/Vialerta/core/files/493979/vialerta_core_db.mwb.tar.gz)

Open with MySQL Workbench

**Comment by erikperin** [comment]
I thought about using a "dumb" database , that doesn't have validations (only standards like  "NOT EMPTY"). All validation will done in the app, including data consistency.

**Comment by soumfilipe** [comment]
> I thought about using a "dumb" database , that doesn't have validations (only standards like "NOT EMPTY"). All validation will done in the app, including data consistency.

Well, This will use less data through the network and less database server's resources. I'm afraid this process result in invalid  or bad data.

**Comment by erikperin** [comment]
@cfilipes But the validation exists... on the app... that validates all the info before INSERTs

**Comment by soumfilipe** [comment]
Yeah, but I'm thinking on a second or third party client communicanting with our db.

**Comment by erikperin** [comment]
@cfilipes In this case, the Core API validates information, no clients can access directly our DB

**Comment by soumfilipe** [comment]
Yep, just making it clear...

**Comment by erikperin** [comment]
Will be more easy for porting... and add new tables... ;)

**Comment by erikperin** [comment]
I make some adjustments on DB...

**Comment by EltonCarreiro** [comment]
Make a dumb db allows you to make heavy migration tasks without any problem. As Erik said, the database MUST be accessed through a core with the necessary validation.
